### ISSUE TITLE
`check_win` does not reset email to original

### ISSUE THREAD

**Issue by muschellij2** [r-lib/devtools#2152]
May be related to https://github.com/r-lib/devtools/issues/1723.

When using `check_win_devel` with the `email` argument, this does not reset after submission.  I see https://github.com/r-lib/devtools/blob/master/R/check-win.R#L61 should indeed copy the original `DESCRIPTION` file over, but the example below shows that's not the case:

``` r
library(devtools)
#> Loading required package: usethis
library(desc)
path = tempfile()
create_package(path, fields = 
                 list(
                   `Authors@R` = 'person("Jane", "Doe", email = "jane@example.com", role = c("aut", "cre"),
                          comment = c(ORCID = "YOUR-ORCID-ID"))',
                   License = "MIT + file LICENSE",
                   Language =  "es"
                 ),
               open = FALSE)
#> ✔ Creating '/var/folders/1s/wrtqcpxn685_zk570bnx9_rr0000gr/T/Rtmpr1oav9/file98c05eb60133/'
#> ✔ Setting active project to '/private/var/folders/1s/wrtqcpxn685_zk570bnx9_rr0000gr/T/Rtmpr1oav9/file98c05eb60133'
#> ✔ Creating 'R/'
#> ✔ Writing 'DESCRIPTION'
#> Package: file98c05eb60133
#> Title: What the Package Does (One Line, Title Case)
#> Version: 0.0.0.9000
#> Authors@R (parsed):
#>     * Jane Doe <jane@example.com> [aut, cre] (<https://orcid.org/YOUR-ORCID-ID>)
#> Description: What the package does (one paragraph).
#> License: MIT + file LICENSE
#> Encoding: UTF-8
#> Language: es
#> LazyData: true
#> ✔ Writing 'NAMESPACE'
#> ✔ Setting active project to '<no active project>'
desc = desc::description$new(file.path(path, "DESCRIPTION"))
desc$get_maintainer()
#> [1] "Jane Doe <jane@example.com>"
devtools::check_win_devel(pkg = path, email = "bob@bob.com")
#> Building windows version of file98c05eb60133 (0.0.0.9000) for R-devel with win-builder.r-project.org.
#>   
   checking for file ‘/private/var/folders/1s/wrtqcpxn685_zk570bnx9_rr0000gr/T/Rtmpr1oav9/file98c05eb60133/DESCRIPTION’ ...
  
✔  checking for file ‘/private/var/folders/1s/wrtqcpxn685_zk570bnx9_rr0000gr/T/Rtmpr1oav9/file98c05eb60133/DESCRIPTION’ (377ms)
#> 
  
─  preparing ‘file98c05eb60133’:
#>    checking DESCRIPTION meta-information ...
  
✔  checking DESCRIPTION meta-information
#> 
  
─  checking for LF line-endings in source and make files and shell scripts
#> 
  
─  checking for empty or unneeded directories
#>    Removed empty directory ‘file98c05eb60133/R’
#> ─  building ‘file98c05eb60133_0.0.0.9000.tar.gz’
#> 
  
   
#> 
#> [05:59 PM (2019-11-26)] Check bob@bob.com for a link to the built package in 15-30 mins.
desc = desc::description$new(file.path(path, "DESCRIPTION"))
desc$get_maintainer()
#> [1] "Jane Doe <bob@bob.com>"
```

<sup>Created on 2019-11-26 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0.9000)</sup>

**Comment by jimhester** [comment]
The problem is the `on.exit()` lower down overwrites it https://github.com/r-lib/devtools/blob/568ac2a11b67cf780972c11122edce7d65f6fddb/R/check-win.R#L85

The second one needs a `add = TRUE` argument as well.

**Comment by jimhester** [comment]
Closed by #2157
